addons/sale/tests/test_sale_order.py

```diff
@@ -19,6 +19,16 @@ class TestSaleOrder(SaleCommon):
     # Those tests do not rely on accounting common on purpose
     #   If you need the accounting setup, use other classes (TestSaleToInvoice probably)
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.salesman_user = cls.env['res.users'].create({
+            'name': 'Salesman',
+            'login': 'salesman_user',
+            'groups_id': [cls.env['ir.model.data']._xmlid_to_res_id('sales_team.group_sale_salesman')],
+        })
+        cls.salesman_user.partner_id.email = 'saleman@test.com'
+
```

Comment:
I believe `self.sale_user` is already available if you need it (SaleCommon inherits from SalesTeamCommon which creates a salesman and salesmanager record)

Reply:
Ah indeed. 🙈 

Thanks I will change that. 🙂